src/components/Chat/ReplayAsIsButton.tsx

```diff
@@ -39,8 +39,7 @@ export const ReplayAsIsButton = ({ replay, conversationId }: Props) => {
     'flex items-center gap-3 rounded border p-3 text-left text-xs',
     {
       'border-blue-500': replay.replayAsIs,
-      'border-gray-400 hover:border-gray-800 dark:border-gray-600 hover:dark:border-gray-200':
-        !replay.replayAsIs,
+      'border-primary hover:border-controls-hover': !replay.replayAsIs,
```

Comment:
in Figma it looks like `border-hover`
![image](https://github.com/epam/ai-dial-chat/assets/20163971/73edec15-12a9-45e1-b746-7d1a6de72f3e)